### ISSUE TITLE
latest sysstat upgrade removes sysstat cron jobs

### ISSUE THREAD

**Issue by ericjunkermann** [void-linux/void-packages#49682]
### Is this a new report?

Yes

### System Info

Void 6.6.21_1 x86_64 AuthenticAMD notuptodate hold rFFFF

### Package(s) Affected

sysstat-12.7.5_1

### Does a report exist for this bug with the project's home (upstream) and/or another distro?

No, this is a packaging issue.

### Expected behaviour

The sysstat cron jobs should continue to run after the upgrade to sysstat-12.7.5_1 .

### Actual behaviour

After the upgrade cron jobs do not run. In fact the file containing them has been removed (from /etc/cron.hourly).  There is a replacement in /etc/cron.d but my scheduling program - fcron - does not read files in that directory.

### Steps to reproduce

1. On a system with an earlier version of sysstat with fcron and sysstat cron jobs running ...
2. Upgrade sysstat to 12.7.5_1
3. Cron jobs no longer run.

**Comment by ericjunkermann** [comment]
This is actually a replacement for #49623, because somebody closed that apparently without noticing that it was about sysstat and only secondarily about fcron in regard to which the only point was the the distribution (or not) of check_system_crontabs.sh .

**Comment by Clos3y** [comment]
Regarding the `fcron` aspect, `check_system_crontabs.sh` overwrites the system crontab (https://github.com/yo8192/fcron/blob/f49f631186c5ef8881845f43d9e58214dfa897b6/script/check_system_crontabs.sh#L9-L11), so I don't think distributing that script is a good idea.

As for `sysstat` specifically, I'm not sure what the solution is. When you look at the sysstat makefile (https://github.com/sysstat/sysstat/blob/443ade20dacee37c4e1cac946212f467a22fcac5/Makefile.in#L625-L632) there are only two options for cron on systemd-less distros: either in `/etc/cron.d` or `/etc/cron.{daily,hourly}`. But the latter causes cron to halt for 50 minutes (#49553).

**Comment by Clos3y** [comment]
> it would have been nice if the sysstat upgrade somehow came with a warning for fcron users.

Yeah I agree: I'll take some blame for not suggesting that in my issue.

So what do you envisage as the resolution to this issue?
